service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwin.java

```diff
@@ -236,7 +183,7 @@ public void updateDesiredProperties(DeviceTwinDevice device) throws UnsupportedO
 
     /**
      * This method replaces desired properties for the specified device. desired properties can be input
-     * via device's setDesiredProperties method.
+     * via device's {@link DeviceTwinDevice.setDesiredProperties(TwinCollection desiredProperties)} method.
```

Comment:
This link doesn't work for me. I need help finding the right syntax. Same for line 199.

Reply:
It should work if you provide the full namespace of DeviceTwinDevice beforehand. Something like {link com.microsoft.azure.sdk.iot.service.devicetwin.DeviceTwinDevice#setDesiredProperties(TwinCollection)}

Reply:
The reason being that you need to disambiguate which deviceTwinDevice class you are referring to

Reply:
that seems to work. I'm surprised about the # symbol before the method.